apps/jsii-docgen/src/docgen/view/api-reference.ts

```diff
@@ -63,10 +64,28 @@ export class ApiReference {
     interfaces = interfaces.filter((item) => !isSkipped(item.docs));
     enums = enums.filter((item) => !isSkipped(item.docs));
 
-    this.constructs = new Constructs(transpile, classes);
+    classes.forEach((c) => {
+      const inflight = getInflight(c.docs);
+      if (inflight) {
+        c.inflightId = MarkdownDocument.sanitize(inflight);
+      }
+    });
+
+    const inflightIds: string[] = classes
+      .map(({ inflightId }) => inflightId)
+      .filter((id) => typeof id === "string") as string[];
```

Comment:
Are these referring to the type names like "@winglang/sdk.cloud.IBucketClient"? If so the name "fqn" is more typical (it stands for Fully Qualified Name and is used in a few other places)
```suggestion
    const inflightFqns: string[] = classes
      .map(({ inflightFqn }) => inflightFqn)
      .filter((fqn) => typeof fqn === "string") as string[];
```